lib/delay-execution.yml

```diff
@@ -14,17 +16,23 @@ rule:
       - B5F85C26D7AA5A1FB4AF5821B6B5AB9B:0x402FA6
   features:
     - or:
-      - api: kernel32.Sleep
-      - api: kernel32.SleepEx
-      - api: kernel32.WaitForSingleObject
-      - api: kernel32.SignalObjectAndWait
-      - api: kernel32.WaitForSingleObjectEx
-      - api: kernel32.WaitForMultipleObjects
-      - api: kernel32.WaitForMultipleObjectsEx
-      - api: kernel32.RegisterWaitForSingleObject
-      - api: WaitOnAddress
-      - api: user32.MsgWaitForMultipleObjects
-      - api: user32.MsgWaitForMultipleObjectsEx
-      - api: NtDelayExecution
-      - api: KeWaitForSingleObject
-      - api: KeDelayExecutionThread
+      - function:
+        - and:
+          - basic block:
+            - or:
+              - api: kernel32.Sleep
+              - api: kernel32.SleepEx
+              - api: kernel32.WaitForSingleObject
+              - api: kernel32.SignalObjectAndWait
+              - api: kernel32.WaitForSingleObjectEx
+              - api: kernel32.WaitForMultipleObjects
+              - api: kernel32.WaitForMultipleObjectsEx
+              - api: kernel32.RegisterWaitForSingleObject
+              - api: WaitOnAddress
+              - api: user32.MsgWaitForMultipleObjects
+              - api: user32.MsgWaitForMultipleObjectsEx
+              - api: NtDelayExecution
+              - api: KeWaitForSingleObject
+              - api: KeDelayExecutionThread
```

Comment:
these should be restricted with `os: windows` while the other block restricted with `os: linux`. we haven't quite gotten to all the rules to specify the os, so when we come across them during updates, we should add those conditions in to make clear which APIs apply to which OSes.